src/tests/bulkConcepts/containers/BulkConceptsPage.test.js

```diff
@@ -421,6 +421,10 @@ describe('Test suite for BulkConceptsPage component', () => {
     const typeWordInSearchField = { target: { name: 'searchInput', value: 'sample' } };
     wrapper.find('#search-concept').simulate('change', typeWordInSearchField);
     expect(wrapper.find('BulkConceptsPage').state().searchInput).toEqual('sample');
+
+    const searchForWordWithFewCharacters = { target: { name: 'searchInput', value: 'y' } };
+    wrapper.find('#search-concept').simulate('change', searchForWordWithFewCharacters);
```

Comment:
Is this test about removing the message?

Reply:
Not exactly, the reason why I added the test was that when the problem I worked on was solved, the coverage dropped so I had to write the test to increase the coverage.

Reply:
So what exactly is this testing?

Reply:
It is meant to test for the instance where the search input is less than three characters.

Reply:
Is that the problem that is being fixed by this ticket?